src/basic/log.c

```diff
@@ -858,8 +858,7 @@ int log_format_iovec(
                 iovec[(*n)++] = IOVEC_MAKE_STRING(m);
 
                 if (newline_separator) {
-                        iovec[*n].iov_base = (char*) &nl;
-                        iovec[*n].iov_len = 1;
+                        iovec[*n] = IOVEC_MAKE((char *)&nl, 1);
                         (*n)++;
```

Comment:
These lines can be merged.

Not matter, but I prefer to add a whitespace between the cast, that is, `(char *) &nl`.

Reply:
I think we can merge this as is. Combining those lines is an option, but the incrementing of the pointer is pretty complex, so I think keeping it separate is nice too.

Reply:
OK.